src/utility/copy.cpp

```diff
@@ -154,6 +166,37 @@ CreateCopyOptions(CopyStmt *copy_stmt, bool *options_valid) {
 	return options_string;
 }
 
+/*
+ * Throws an error if a rewritten raw statement returns an unexpected number of
+ * queries (i.e. not just a single query). This is taken from Postgres its
+ * BeginCopyTo function.
+ */
+void
+check_rewritten(List *rewritten) {
```

Comment:
nit
```suggestion
CheckRewritten(List *rewritten) {
```
(to keep consistency w/ other functions in this file - unless it's to keep the Postgres original casing?)

Also it seems this function isn't used?